force-app/main/default/classes/AFFL_AccRecordType_TEST.cls

```diff
@@ -189,6 +208,81 @@ public with sharing class AFFL_AccRecordType_TEST {
         updateAdminAccRecordTypeBulk();
     }
 
+    /*
+     * Ensure redundant affiliaition RT mappings is surfaced as an error upon contact update.
+     * Test Scenario: T-3939010
+     */
+    @isTest
+    private static void invalidRedundantAffiliationRTMappingCheck_FieldLabels() {
+        setupWithFieldLabels();
+        // Insert redundant affiliation mapping
+        insert new Affl_Mappings__c(Name = 'Household Account', Account_Record_Type__c = 'HH_Account');
+
+        try {
+            changePrimaryAffiliationRecordType();
+        } catch (Exception ex) {
```

Comment:
This will swallow nearly any exception. We should limit this exception to the specific type expected in order to make the test valid only under the scenarios we expect.